index.html

```diff
@@ -4026,23 +4026,23 @@ <h2><span class="chunk">cICP</span>
 
 
 <p>Coding-Independent Code Points are defined in
-  <a href="#2-ITU-T-H.273"><span class="NormRef">[ITU-T H.273]</span></a> and 
-  are typically embedded in video content and streams to define an explicit 
+  <a href="#2-ITU-T-H.273"><span class="NormRef">[ITU-T H.273]</span></a> and
+  are typically embedded in video content and streams to define an explicit
   video rendering. The most common renderings for video imagery are defined in
   <a href="#2-ITU-R-BT.709"><span class="NormRef">[ITU-R BT.709]</span></a>
   and <a href="#2-ITU-R-BT.2100"><span class="NormRef">[ITU-R BT.2100]</span></a></p>
 
   <p>The cICP chunk contains four decimal values corresponding to
-  the colour primaries, transfer function, matrix coefficients and video signal 
+  the colour primaries, transfer function, matrix coefficients and video signal
   range flag for the source imagery. </p>
 
-  <p>Note: In Coding-Independent Code Points 
-    <a href="#2-ITU-T-H.273"><span class="NormRef">[ITU-T H.273]</span></a>, the 
+  <p>Note: In Coding-Independent Code Points
+    <a href="#2-ITU-T-H.273"><span class="NormRef">[ITU-T H.273]</span></a>, the
     third video coding point defines matrix coefficients which are used to describe
-    conversions from other colour representations to RGB. PNG image samples are 
-    explicitly RGB, so the matrix coefficients value must be set to "0" which is 
-    labeled in <a href="#2-ITU-T-H.273"><span class="NormRef">[ITU-T H.273]</span></a> 
-    as "RGB Identity". With PNG, decoders will always assume a matrix coefficient 
```

Comment:
Same here. Was removing the trailing space intentional?

(We should probably run this doc through a formatter at some point.)